src/lib/scene-internal.ts

```diff
@@ -455,38 +460,41 @@ export class SceneInternal implements Renderer {
       throw new Error('Could not find REGL');
     }
 
+    this.canvas = document.createElement('canvas');
```

Comment:
Creating the canvas explicitly prevents regl from adding its own size change observers, which it only does when it creates the canvas.